tools/rollup.config.dev.js

```diff
@@ -48,11 +48,9 @@ module.exports = {
     }),
     babel({
       exclude: ['node_modules/**'],
-      plugins: ['external-helpers'],
```

Comment:
Any reason for dropping this plugin?

Reply:
@joshblack https://github.com/rollup/rollup-plugin-babel#helpers is the best resource. In short, Rollup requires using `babel-plugin-external-helpers` with Babel 6, but not the case for Babel 7.